source/2020-07-24-the-ember-times-issue-157.md

```diff
@@ -25,14 +27,10 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Tweaking UI behavior with Ember Data queries 📝](https://codingitwrong.com/2020/06/23/ember-list.html)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[Josh Justice (@CodingItWrong)](https://github.com/codingitwrong) has a new [blog post](https://codingitwrong.com/2020/06/23/ember-list.html) about Ember Data, his favorite part of Ember.js. Josh writes about building a todo list application, [Surely](https://github.com/CodingItWrong/surely-ember). He touches on querying with JSON:API, the basics of working with the model, sorting with Ember Data, the nuances of reloading, and more. Check out [Tweaking UI behavior with Ember Data queries] and the accomanying [GitHub repo](https://github.com/CodingItWrong/surely-ember) to learn more about Ember Data! 
```

Comment:
typo: "accomanying" -> "accompanying"

Reply:
The phrase "to learn more about Ember Data" sounds addressed to folks that might not be very familiar with Ember Data at all. If that seems like the best focus for within the newsletter, that is all good.

But my own intent in the newsletter was to address someone where I am: one step _beyond_ the basics. I know the fundamentals of Ember Data works, and this blog post addresses how to put those pieces together to create a better UX than the default. A _slightly_ different phrase that captures this might be "…to learn more about putting Ember Data to use." Feel free to use that if you like.

Reply:
@CodingItWrong 👍 totally agree! Thank you for catching the typo and the suggestion!